plugins/vendor/zoom/lib/api/webPage/KalturaZoomRegistrationPage.html

```diff
@@ -161,96 +74,190 @@
             });
         }
         function  onSuccess() {
-            $('#submitError').hide();
-            $('#submitOK').show();
+            alert("Saved Successfully");
         }
         function onFail(errorMessage) {
-            $('#submitOK').hide();
-            $('#submitError').show();
-            $('#submitError').html(errorMessage);
+            alert("Data was Not Saved.\n"+errorMessage);
+        }
+        function onLoad()
+        {
+            console.log(partnerId);
+            if(!partnerId)
+            {
+                return;
+            }
+            setDefaultValues();
+            $.ajax({
+                url: getUrl,
+                type: 'post',
+                data: {"ks": ks, 'accountId': accountId ,'partnerId':partnerId },
+                success: function (res) {
+                    console.log(res);
+                    if (res) {
+                        if (res.objectType == 'KalturaAPIException') {
+                            alert("Error loading page.\n"+res.message);
+                        } else {
+                            //   onSuccess();
+                            console.log(res);
```

Comment:
you already printed res on line 95 and it was not changed since then
also need to fix { } indentation